shim.go

```diff
@@ -32,33 +32,33 @@ var sigIgnored = map[syscall.Signal]bool{
 
 type shim struct {
 	containerId string
-	pid         uint32
+	execId      string
 
 	ctx   context.Context
 	agent *shimAgent
 }
 
-func newShim(addr, containerId string, pid uint32) (*shim, error) {
+func newShim(addr, containerId, execId string) (*shim, error) {
 	if agent, err := newShimAgent(addr); err != nil {
 		return nil, err
 	} else {
 		return &shim{containerId: containerId,
-			pid:   pid,
-			ctx:   context.Background(),
-			agent: agent}, nil
+			execId: execId,
+			ctx:    context.Background(),
+			agent:  agent}, nil
```

Comment:
You have introduced trailing white spaces on both lines, please remove them.

Reply:
There are no trailing whitespaces at the end of the three lines.

Reply:
Not at the end, it's between `ctx:` and `context.Background(),` and between `agent:` and `agent`.
If you run `go fmt` or `gofmt -s`, it should be solved automatically.

Reply:
It was actually added by `go fmt`, to align the three assignment.

Reply:
Ohhhh my bad, you're right...